my-dsl.md

```diff
@@ -2,37 +2,105 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The language is called [ACSL](http://frama-c.com/acsl.html), but the tools to
+work with it are collectively [Frama-C](http://frama-c.com/what_is.html) and a
+variety of [plugins](http://frama-c.com/plugins.html).
 
 # Domain
 _Describe the language's domain in five words._
 
+Formal specifications for C code
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+ACSL is not exactly an executable language, _per se_, and what happens depends
+very much on the plugins used. It is a specification language, which can be used
+to add assert statements, prove functions correct, prove loops terminate, or
+other things. This is done by a Frama-C plugin or plugins reading the
+declarative specifications to figure out what it needs to do (e.g., what
+assertions to add, what test cases to prove, etc.), and then attempting to do
+it (e.g., output new C code with additional assert statements, give cases to a
+theorem prover, etc.).
+
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+ACSL is almost purely domain-specific, but can encode arbitrary mathematical
+specifications and therefore Turing-complete computations. I can’t see why
+anybody would use ACSL as a general-purpose language, but it is theoretically
+possible. In this case, I would classify ACSL as slightly more domain-specific
+than the XSLT example that Fowler mentions. It can theoretically do anything,
+but it is much more focussed on formal specification of other code than XSLT is
```

Comment:
Good! Even though a language can do anything doesn't mean it should or was designed to.